_config.yml

```diff
@@ -16,9 +16,16 @@ sass:
 host:           localhost # same as default value '127.0.0.1' but reads better
 
 # Links
-main:           "https://getbootstrap.com"
-expo:           "https://expo.getbootstrap.com"
-themes:         "https://themes.getbootstrap.com"
+current_version: "4.3.1"
+docs_version:    "4.3"
+
+main:            "https://getbootstrap.com/"
```

Comment:
Reminder to self, double check all these; we need to adapt the code to not have double slashes or missing ones